test/functional/test.sh

```diff
@@ -21,6 +21,34 @@ do
     fi
 done
 
+declare -a sites_convert=("test_site_convert")
+
+for site in "${sites_convert[@]}"
+do
+    # print site name
+    echo
+    echo "Running $site tests"
+
+    # convert site
+    node ../../index.js init "$site" -c
+
+    # build site
+    node ../../index.js build "$site"
+
+    # run our test script to compare html files
+    node testUtil/test.js "$site" "test_site_convert_expected"
```

Comment:
This shouldn't be hard code, it defeats the purpose of having the `sites_convert` array in the first place.